<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>780</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="uploadManifestPage">
    <widget class="QPushButton" name="uploadManifestButton">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>150</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Manifest</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="balanceLoadPage">
    <widget class="QPushButton" name="balanceButton">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>131</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Balance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>131</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Load/Unload</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loadContainerPage">
    <widget class="QLabel" name="enterContainersLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>491</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the containers you would like to load on the ship:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loadContainerLineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="doneLoadingButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="unloadContainerPage">
    <widget class="QPushButton" name="unloadButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Unload</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loadingPage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skipPageLoadingButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>SkipPage</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="viewStepPage">
    <widget class="QPushButton" name="makeNoteButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Make Note To Log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextStepButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>290</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="currStepPageLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Step: </string>
     </property>
    </widget>
    <widget class="QLabel" name="estimatedTimeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Estimated Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="moveContainerLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>541</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="currUserLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="debugModeCheckBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Debug Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTextEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>162</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
